docs/source/rdump.rst

```diff
@@ -0,0 +1,115 @@
+rdump
+-----
+
```

Comment:
it's throwing off that the first line on rdump is not about rdump. 
maybe add an introduction line like the one in the tutorial
```suggestion
`rdump` is used to process, filter and format target-query results.
```